src/svgSprite.ts

```diff
@@ -542,5 +542,17 @@ export const svgSprite = (): string => `
       </clipPath>
     </defs>
   </symbol>
+
+  <symbol viewBox="0 0 18 18" id="svg-editor">
```

Comment:
Could you use '0 0 24 24' box for all svgs (to make all the icons have the standard view box)?

Reply:
https://www.figma.com/file/LXl9BAJSwtgWuzkxqOqvX8/ML1-Design-System?node-id=53015%3A36793&t=SAqJtQPlTsvKXOIb-0